main.py

```diff
@@ -1,7 +1,39 @@
+from django.contrib.contenttypes.fields import GenericRel  # noqa: F401
+
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    genres = ["Western", "Action", "Dramma"]
+
+    for name in genres:
+        Genre.objects.create(
+            name=name
+        )
+    actors = [
+        ("George Klooney"),
+        ("Kianu Reaves"),
+        ("Scarlett Keegan"),
+        ("Will Smith"),
+        ("Jaden Smith"),
+        ("Scarlett Johansson"),
```

Comment:
The actor names are currently enclosed in parentheses, which makes them tuples with a single string element. This will cause an error when trying to split the name in line 27. Remove the parentheses to make them simple strings.